site/en/_book.yaml

```diff
@@ -150,10 +150,17 @@ upper_tabs:
       - heading: Optimizing Bazel
       - title: Best practices
```

Comment:
this page and the next do not really fit under "Optimizing Bazel"

@philomathing any idea how to group these two?

Reply:
On a different note, I noticed that the headings are not visible on the page https://bazel.build/docs
This leads to some confusing entries:

<img width="1489" alt="Screenshot 2022-12-16 at 16 25 27" src="https://user-images.githubusercontent.com/4410333/208131371-23f5616b-897a-427d-8586-865dd770c3dd.png">

Is it worth opening an issue for this?

Reply:
Yeah, let's file an issue for this

Reply:
https://github.com/bazelbuild/bazel/issues/17049